.packit.yaml

```diff
@@ -0,0 +1,59 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+packit_instances: ["stg"]
+upstream_project_url: https://github.com/fedora-copr/copr.git
+
+packages:
+
+  python:
+    downstream_package_name: python-copr
+    upstream_package_name: copr
+    paths: 
+      - ./python
```

Comment:
My feedback on this
- good this is an array, I suppose that change on these paths trigger the actual build, right
- being able to "deny-list" some sub-directories (somehow) would be nice

Reply:
Actually this is a list but only the first entry, at the moment, is used as root for building the sub-package. I am waiting for an example where a list is really needed, if you have one please let me know :)

Regarding the deny list, I might be wrong, but I think @csomh already has something for it.